components/txn_types/src/types.rs

```diff
@@ -253,6 +253,24 @@ pub enum MutationType {
     Other,
 }
 
+/// A row mutation.
+#[derive(Debug, Clone)]
+pub enum RawMutation {
+    /// Put `Value` into `Key` with TTL. The TTL will overwrite the existing TTL value.
+    Put((Key, Value), Option<u64>),
```

Comment:
Would you refactor this struct and make it more readable? Like:
```rust
pub enum RawMutation {
	Put({ key: Key, value: Value, ttl: Option<u64>)
    ...
}
```

Reply:
I think this is adopted for `Mutation`. But anyway, I think this is what you may want to say:

```
pub enum RawMutation {
	Put { key: Key, value: Value, ttl: Option<u64> }
    ...
}
```

Reply:
Yes. Sorry for the extra '('

Reply:
committed